### PR TITLE
Expand the bungee module

### PR DESCRIPTION
**Pull request by STG-Allen**
This PR aims to add all of the same functionality from the Velocity module to the Bungee module.

### CODE REVIEW

File: cloud-minecraft/cloud-bungee/src/main/java/cloud/commandframework/bungee/BungeeCommandManager.java [hunk 1]
```diff
@@ -46,21 +57,33 @@
      * @param commandExecutionCoordinator  Coordinator provider
      * @param commandSenderMapper          Function that maps {@link CommandSender} to the command sender type
      * @param backwardsCommandSenderMapper Function that maps the command sender type to {@link CommandSender}
-     * @throws Exception If the construction of the manager fails
```
Comment (STG-Allen) [inline cloud-minecraft/cloud-bungee/src/main/java/cloud/commandframework/bungee/BungeeCommandManager.java:49]: Double-check the validity of the removal.

**Comment by STG-Allen** [comment]
image

Commands in action.

**Comment by STG-Allen** [comment]
Todo: add argument annotation parsing

File: cloud-minecraft/cloud-bungee/src/main/java/cloud/commandframework/bungee/BungeeCommandManager.java [hunk 2]
```diff
@@ -46,21 +57,33 @@
...
             final @NonNull Function<@NonNull C, @NonNull CommandSender> backwardsCommandSenderMapper
-    )
-            throws Exception {
+    ) {
         super(commandExecutionCoordinator, new BungeePluginRegistrationHandler<>());
         ((BungeePluginRegistrationHandler<C>) this.getCommandRegistrationHandler()).initialize(this);
         this.owningPlugin = owningPlugin;
         this.commandSenderMapper = commandSenderMapper;
         this.backwardsCommandSenderMapper = backwardsCommandSenderMapper;
+
```
Comment (jmanpenilla) [inline cloud-minecraft/cloud-bungee/src/main/java/cloud/commandframework/bungee/BungeeCommandManager.java:73]: The PlayerArgument and ServerArgument parsers should be registered with the parser registry here, similar to [this](https://github.com/Incendo/cloud/commit/4995b4c2d7de476b9d23c1c97cd3751bc75550b6) commit

Comment (STG-Allen) [reply cloud-minecraft/cloud-bungee/src/main/java/cloud/commandframework/bungee/BungeeCommandManager.java:78]: Added, working on fixing the branch now.

**Review by jmanpenilla — APPROVED** [review]

**Pull request merged**